homework/list.js

```diff
@@ -1,44 +1,49 @@
 'use strict';
 
-(function() {
+(function(exports) {
+var ListManager = function(){
+	this._listNoteContent = [];
+	this._wrapper = null;	
+};
 
-  var _listNoteContent = [];
-  var _wrapper = document.querySelector('#note-list-wrapper');
-
-  function start() {
-    fetchList(function(data) {
-      updateList(data);
-      drawList();
-      preloadFirstNote();
-    });
-    window.addEventListener('click', function(event) {
-      onNoteOpen(event);
-    });
-  }
-
-  function onNoteOpen(event) {
+ListManager.prototype = {
+	
+  start() {
+	this._wrapper = document.querySelector('#note-list-wrapper');
+    this.fetchList()
+	.then((function(data) {
+      this.updateList(data);
+      this.drawList();
+      this.preloadFirstNote();
+    }).bind(this));
+    window.addEventListener('click', (function(event) {
```

Comment:
It's a convention to prefix method with `on` for the corresponding function, so it should be renamed as 'onClick'. And, to set `this` as the listener is better than an anonymous function.